src/org/labkey/test/components/DomainDesignerPage.java

```diff
@@ -152,5 +158,8 @@ DomainFormPanel domainFormPanel(String title) // for situations with multiple do
             return new DomainFormPanel.DomainFormPanelFinder(getDriver()).withTitle(title)
                     .timeout(WAIT_FOR_JAVASCRIPT).findWhenNeeded(this);
         }
+
+        public final Checkbox selectAll = new Checkbox(Locator.tagWithAttributeContaining("input", "id", "domain-select-all-checkbox")
+                .findWhenNeeded(this));
```

Comment:
Is this the correct scope? I think there could be multiple "select all" checkboxes on the page for multi-domain designers (such as assay result, run, and data domains).
I think the select all functionality belongs in `DomainFormPanel`

Reply:
Changed the location